### DISCUSSION TITLE
Herd doesn't seem to work with Git Bash Terminal

### DISCUSSION THREAD

**Discussion by frankliniwobi** [beyondcode/herd-community#697] [Q&A] [answered]
I've been using git bash on my windows pc for the longest time now, no issues, but after I installed herd, I've discovered that the `herd` command doesn't work on git bash, I've switched to the default Windows power shell and it's working perfectly but I don't like Windows power shell, it's not just good enough for me, so please is there a way to get herd to work on git bash? Or can anyone please recommend any terminal that is as good as git bash, works with herd and supported by Windows, I've been searching, I'd appreciate any help I can get, thank you 🙏

**Comment by mechelon** [comment, marked as answer]
If you want to use the Herd binaries in Git Bash, add these aliases to your `.bash_profile`:

```bash
alias php="php.bat"
alias herd="herd.bat"
alias laravel="laravel.bat"
alias composer="composer.bat"
```

We'll update the documentation regarding that soon (and will also try to add these lines automatically during onboarding for new installs).

**Reply by spitfire64** [reply]
My .bash_profile now looks like this: 
```
# generated by Git for Windows
test -f ~/.profile && . ~/.profile
test -f ~/.bashrc && . ~/.bashrc
alias php="php.bat"
alias herd="herd.bat"
alias laravel="laravel.bat"
alias composer="composer.bat"

```
and my .bashrc like this: 
`
pa=php artisan
`
But the shorthand 'pa' doesn't work.

**Reply by bhekor** [reply]
Thank you for this, I almost reset my pc because of this. Because I've uninstalled Herd, but it didn't clear the aliases. I didn't know it created aliases for me, I feel it's unnecessary.

**Reply by Mohamed-G2021** [reply]
> If you want to use the Herd binaries in Git Bash, add these aliases to your `.bash_profile`:
> 
> ```shell
> alias php="php.bat"
> alias herd="herd.bat"
> alias laravel="laravel.bat"
> alias composer="composer.bat"
> ```
> 
> We'll update the documentation regarding that soon (and will also try to add these lines automatically during onboarding for new installs).

worked for me 
thanks for the reply

**Comment by jcrben** [comment]
nvm found them in the ~/.bash_profile

~~@mechelon  where do these get added? installed herd and now I've noticed I have these new aliases but I want to remove them~~

See also: [[Bug]: uninstall docs for Windows missing the entries added to .bash_profile #930](https://github.com/beyondcode/herd-community/issues/930)
